### PR TITLE
Make grid pattern appear

### PR DESCRIPTION
**Pull request by alflennik**
Address issue #102.

### CODE REVIEW

**Comment by richnoah** [comment]
@alflennik I see Grid present and the content looks good.

File: content/patterns.md [hunk 1]
```diff
@@ -207,7 +207,21 @@ WAI-ARIA supports two types of checkbox widgets: dual-state allows the user to t
...
-                 
+                
+                <li class="tile tile-grid">
+                  <a href="{{ '/ARIA/apg/patterns/grid/' | relative_url }}">
+                    <h2 class="tile-name">
+                      <img 
+                        src="{{ '/content-images/wai-aria-practices/img/grid.svg' | relative_url }}" 
+                        alt=""
+                      >
+                      <span>Grids : Interactive Tabular Data and Layout Containers</span>
```
Comment (howard-e) [inline content/patterns.md:218]: This pattern should come before `Landmarks`

**Review by howard-e — CHANGES_REQUESTED** [review]
@alflennik looks good to me other than the comment relating to the alphabetical order on the patterns page. Good stuff!

Another thing of note is on [Data Grid Examples Page, Example 2: Sortable Data Grid With Editable Cells](https://deploy-preview-112--wai-aria-practices2.netlify.app/aria/apg/example-index/grid/datagrids), there appears to be some conflicting CSS here which is forcing the edit icon over the column line for its respective cell. But that's outside the purpose of this PR and I believe there may be an over-arching task to account for these kind of task? If not, it could be filed in a separate issue.

File: content/patterns.md (hunk 1 above)
Comment (alflennik) [reply content/patterns.md:218]: What a great catch! I wouldn't have noticed in a million years.

**Comment by alflennik** [comment]
@howard-e I decided to go ahead and fix the offending example.

**Comment by mcking65** [comment]
Thank you @alflennik!!!

**Pull request merged**